### DISCUSSION TITLE
Players in China need more startup parameters to watch replay

### DISCUSSION THREAD

**Discussion by xqm32** [fraxiinus/ReplayBook#132] [Troubleshooting]
Players in China need to add arguments `-Region=TENCENT -Locale=zh_CN -SkipRads -SkipBuild` to watch replay without bug, or there may be some messy codes.

I have made a batch script (https://github.com/xqm32/LOLReplay) to do this, but it's not easy to use like ReplayBook.

**Comment by fraxiinus** [comment]
Good to know! It should be possible to add these arguments to ReplayBook:

Make sure the region is set to 'zh_CN':
![image](https://user-images.githubusercontent.com/8365233/147807267-d163a873-d9c7-420e-87bb-9432a617a887.png)

Press the pencil icon on the "other arguments" box. Then add "-Region=TENCENT" to the list. Be aware of the quotation marks.
![image](https://user-images.githubusercontent.com/8365233/147807316-31b10fbb-328b-41e7-acda-b2121858ea2c.png)

Let me know if this works!

**Reply by JerryLiew** [reply]
it should works . that is what i did.

**Reply by xqm32** [reply]
It works. Thank you! The argument `-Region=TENCENT` seems not necessary. I deleted `-Region=TENCENT` and it also works.
